### PR TITLE
Update ak bills scraper

### PR DESCRIPTION
**Pull request by schneidy**
Updated Alaska (AK) Bill Scraper. Vote details still don't currently work, but basic vote information is still added.

### CODE REVIEW

File: openstates/ak/bills.py [hunk 1]
```diff
@@ -71,39 +71,41 @@ def scrape(self, chamber=None, session=None):
...
-                '.asp?session=%s&bill1=%s1&bill2=%s9999'
-            ) % (session, abbr, abbr)
-            doc = lxml.html.fromstring(self.get(bill_list_url).text)
-            doc.make_links_absolute(bill_list_url)
-            for bill_link in doc.xpath('//table[@align="center"]//tr/td[1]//a'):
-                bill_url = bill_link.get('href')
-                bill_id = bill_link.text.replace(' ', '')
-                yield from self.scrape_bill(chamber, session, bill_id, bill_type,
-                                            bill_url)
+        bill_list_url = (
```
Comment (jamesturk) [inline openstates/ak/bills.py]: for new code, let's favor Python 3 style string formatting

either 'foo {}'.format(session) or f'foo {session}'  :)

File: openstates/ak/bills.py [hunk 2]
```diff
@@ -198,109 +205,159 @@ def scrape_bill(self, chamber, session, bill_id, bill_type, url):
...
+                elif "E" in vote_count or "A" in vote_count:
+                    other += int(vote_count[1:])
+
+
+        vote = VoteEvent(
+                bill=bill,
+                start_date=act_date.strftime('%Y-%m-%d'),
+                chamber=act_chamber,
+                motion_text=action,
+                result='pass' if yes > no else 'fail',
```
Comment (jamesturk) [inline openstates/ak/bills.py]: is this our best proxy for passage?  it is often incorrect so let's double check that there's no passage indicator on the page

File: openstates/ak/bills.py [hunk 3]
```diff
@@ -198,109 +205,159 @@ def scrape_bill(self, chamber, session, bill_id, bill_type, url):
...
-        votes_data = zip(votes_text[1::2], votes_text[2::2])
+        yield vote
 
-        iVoteOnPage = 0
+        
+        # print(action)
 
-        # Process each.
-        for motion, text in votes_data:
+        # Dan attempt at pulling out the vote information
```
Comment (jamesturk) [inline openstates/ak/bills.py]: could you post a few links to the pages this is scraping so I can review them and offer some suggestions?  (both the metadata and the journal entries)

Comment (schneidy) [reply openstates/ak/bills.py]: For [HB14](https://www.akleg.gov/basis/Bill/Detail/31?Root=HB14), here are 3 different votes:

- https://www.akleg.gov/basis/Journal/Pages/31?Chamber=H&Bill=HB%20%2014&Page=00727#0727
- https://www.akleg.gov/basis/Journal/Pages/31?Chamber=H&Bill=HB%20%2014&Page=00737#0737
- https://www.akleg.gov/basis/Journal/Pages/31?Chamber=H&Bill=HB%20%2014&Page=00739#0739

The last two of the above links lead to the same journal page with the detailed vote information, but are two different votes for the bill.

**Comment by jamesturk** [comment]
Oof OK, yeah that's not a fun one.  Let's merge this in two pieces, bills
(with versions, actions, etc.) first, then we'll add votes in a second PR.

File: openstates/ak/bills.py (hunk 1 above)
Comment (schneidy) [reply openstates/ak/bills.py]: Fixed.

File: openstates/ak/bills.py (hunk 2 above)
Comment (schneidy) [reply openstates/ak/bills.py]: Fixed. Now checking action to see if passed or failed.

**Pull request merged**